minimesos/src/main/java/com/containersol/minimesos/mesos/Registrator.java

```diff
@@ -0,0 +1,50 @@
+package com.containersol.minimesos.mesos;
+
+import com.containersol.minimesos.cluster.MesosCluster;
+import com.containersol.minimesos.config.RegistratorConfig;
+import com.containersol.minimesos.container.AbstractContainer;
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import com.github.dockerjava.api.model.Bind;
+
+public class Registrator extends AbstractContainer {
```

Comment:
I think referring to shortcomings in other places is a weak excuse for introducing new ones. `Registrator` is a very generic word. What is this container used for?